src/rng.rs

```diff
@@ -0,0 +1,19 @@
+//! Random Number Generator Interface
+
+use generic_array::{ArrayLength, GenericArray};
+use nb;
+
+/// Nonblocking stream of random bytes.
+// reason: No implementation or users yet
```

Comment:
Shouldn't there be a `cfg` gate instead of a comment?

Reply:
I've got `#[cfg(feature="unstable")]` on the entire `rng` mod -- I'm happy to put it here as well, or to move this comment to that point. What would you prefer?

Reply:
Unstable? ;)

For consistency reasons I'd prefer to have it right in front of the trait definition instead of this comment.

Reply:
> unstable

Whoops. Got it right in the PR, at least =P

I'll put it there

Reply:
> unstable

Whoops. Got it right in the PR, at least =P

I'll put it there

(used the wrong account for the first comment)